scikit_mol/fingerprints.py

```diff
@@ -45,7 +64,11 @@ def _transform_mol(self, mol):
         return arr
 
     def fit(self, X, y=None):
-        """Included for scikit-learn compatibility, does nothing"""
+        """Included for scikit-learn compatibility
+
+        Also sets the column prefix for use by the transform method with dataframe output.
+        """
+        self._get_column_prefix()
```

Comment:
This is not doing anything as the return is simply discarded???

Reply:
Yes, I added it there when I was developing the code, then I forgot about it.

Done in https://github.com/EBjerrum/scikit-mol/pull/39/commits/49d39d0aff92f2deca783905c50e4c6e02625f97